### PR TITLE
IOTA Palantír Messaging (IPM) implementation

### PR DESCRIPTION
**Pull request by yillkid**
This is a spike to implement IOTA Palantír Messaging (IPM)[1] in TangleID new_claim[2] and get_all_claims_in_channel[3] API commands. Just add the key "addr" and "next_addr" keys in new_claim API request body for enabling the function and use get_all_claims_in_channel to query all claims in one channel, refer the API mamual[2][3] or test script file "tests/tangleid/new_claim_ipm.sh",  "tests/tangleid/get_all_claims_in_channel.sh" for detail.
The other hand, one complete workflow demonstration could be found on here[4].

[1] https://blog.florence.chat/introducing-iota-palant%C3%ADr-messaging-ipm-c599ed6d2191
[2] https://hackmd.io/s/HJyzQvF1z
[3] https://hackmd.io/s/ry-8g8nNQ
[4] https://github.com/yillkid/TangleID-IPM-sample

### CODE REVIEW

File: tests/tangleid/get_all_claims_in_channel.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (jserv) [inline tests/tangleid/get_all_claims_in_channel.sh]: Why would you modify TangleID extension test cases?

**Comment by jserv** [comment]
Since you are implementing IPM, there should be an entry to technical description inside source tree.

File: extensions/tangleid/main.py [hunk 2]
```diff
@@ -175,3 +181,20 @@ def new_group(data):
     response = send_transfer(tag, json.dumps(data), address, 0, dict_tips, debug=0)
 
     return str(response)
+
+def get_all_claims_in_channel(data):
```
Comment (jserv) [inline extensions/tangleid/main.py]: Don't touch TangleID extension while implementing core routines about IPM. Instead, you shall get it done as the top level service.

**Comment by jserv** [comment]
Use Markdown syntax to describe what you managed to express in pull request body.

**Comment by jserv** [comment]
Also, I don't believe the complete implementation of IPM can be landed within the few lines of code.

**Comment by jserv** [comment]
@yillkid, you shall explain why this pull request was obsolete.

**Comment by yillkid** [comment]
Will create a new pull request after code refectory and new documentation.

**Pull request closed without merging**